purchases/src/test/java/com/revenuecat/purchases/paywalls/components/common/ComponentOverridesTests.kt

```diff
@@ -9,14 +9,15 @@ import org.junit.Test
 import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
+import org.robolectric.ParameterizedRobolectricTestRunner
 
 @RunWith(Enclosed::class)
 internal class ComponentOverridesTests {
 
     // This tests deserialization of ComponentOverrides containing PartialTextComponent and PartialImageComponent, just
     // to make sure deserialization of generics works as expected.
 
-    @RunWith(Parameterized::class)
+    @RunWith(ParameterizedRobolectricTestRunner::class)
```

Comment:
I'm adding a test where a log gets called. It was failing with the junit runner.